charts/openmetadata/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.28
+version: 0.0.29
```

Comment:
Bump Versions to `0.0.29` for charts/openmetadata and charts/deps. We love to sync here :)